src/generators/Sequence.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Created by chetanv on 01/06/16.
+ */
+
+import Generator from './Generator';
+
+class Sequence extends Generator {
+  static sequences = {};
```

Comment:
I used static to help with a couple of specs: `'initialises the sequence for id'` and `'does not reset the sequence for id'`. 

We can use module scope as well, but then we'll need to either add an instance method to `Sequence` to return current value (for its `id`) or a static method that returns sequence value for the given `id`. Just kept it simple :).